src/utils/types.ts

```diff
@@ -10,6 +10,17 @@ export interface VenueType {
 
 export interface OfferType {
   venue: VenueType;
+  stocks: StockType[];
+}
+
+export interface StockType {
```

Comment:
```suggestion
export interface Stock {
```
Pourquoi préciser `Type` ?

Reply:
Car il était précisé `VenueType`, `OfferType` et `ResultType` pour les autres types, donc j'ai gardé le même principe!
Je le change pour tout du coup tu penses?